redis/redis.go

```diff
@@ -211,3 +211,102 @@ type LatencyHistory struct {
+	ClientProtected             ClientFlags = 1 << 28 /* Client should not be freed for now. */
+	ClientExecutingCommand      ClientFlags = 1 << 29 /* Indicates that the client is currently in the process of handling a command. usually this will be marked only during call() however, blocked clients might have this flag kept until they will try to reprocess the command. */
+	ClientPendingCommand        ClientFlags = 1 << 30 /* Indicates the client has a fully parsed command ready for execution. */
+	ClientTracking              ClientFlags = 1 << 31 /* Client enabled keys tracking in order to perform client side caching. */
+	ClientTrackingBrokenRedir   ClientFlags = 1 << 32 /* Target client is invalid. */
+	ClientTrackingBCAST         ClientFlags = 1 << 33 /* Tracking in BCAST mode. */
+	ClientTrackingOptIn         ClientFlags = 1 << 34 /* Tracking in opt-in mode. */
+	ClientTrackingOptOut        ClientFlags = 1 << 35 /* Tracking in opt-out mode. */
+	ClientTrackingCaching       ClientFlags = 1 << 36 /* CACHING yes/no was given, depending on optin/optout mode. */
+	ClientTrackingNoLoop        ClientFlags = 1 << 37 /* Don't send invalidation messages about writes performed by myself.*/
+	ClientInToTable             ClientFlags = 1 << 38 /* This client is in the timeout table. */
+	ClientProtocolError         ClientFlags = 1 << 39 /* Protocol error chatting with it. */
+	ClientCloseAfterCommand     ClientFlags = 1 << 40 /* Close after executing commands and writing entire reply. */
+	ClientDenyBlocking          ClientFlags = 1 << 41 /* Indicate that the client should not be blocked. currently, turned on inside MULTI, Lua, RM_Call, and AOF client */
+	ClientReplRDBOnly           ClientFlags = 1 << 42 /* This client is a replica that only wants RDB without replication buffer. */
+	ClientNoEvict               ClientFlags = 1 << 43 /* This client is protected against client memory eviction. */
+	ClientAllowOOM              ClientFlags = 1 << 44 /* Client used by RM_Call is allowed to fully execute scripts even when in OOM */
+	ClientNoTouch               ClientFlags = 1 << 45 /* This client will not touch LFU/LRU stats. */
+	ClientPushing               ClientFlags = 1 << 46 /* This client is pushing notifications. */
+	ClientModuleAuthHasResult   ClientFlags = 1 << 47 /* Indicates a client in the middle of module based auth had been authenticated from the Module. */
+	ClientModulePreventAOFProp  ClientFlags = 1 << 48 /* Module client do not want to propagate to AOF */
+	ClientModulePreventReplProp ClientFlags = 1 << 49 /* Module client do not want to propagate to replica */
+	ClientReExecutingCommand    ClientFlags = 1 << 50 /* The client is re-executing the command. */
+	ClientReplRDBChannel        ClientFlags = 1 << 51 /* Client which is used for RDB delivery as part of RDB channel replication */
+	ClientInternal              ClientFlags = 1 << 52 /* Internal client connection */
+)
+
+// Client is redis-server ClientInfo
```

Comment:
suggestion: use sentences so end with a full stop.

```suggestion
// Client represents a redis-server client.
```